src/ssr-carousel-track.vue

```diff
@@ -144,7 +147,7 @@ export default
 
 	# Render the track and slotted slides
 	render: (create) ->
-		create 'div',
+		create @trackHTMLElement,
```

Comment:
We could possibly make this not require a new prop by checking if the firs slide returned by `makeSlides()` has an element of `li`.  Can you console.log it and see if has a `type` or `element` property or something like that so we can try and auto-determine this?